docs/porting/connectivity/CellularInterface.md

```diff
@@ -2,6 +2,8 @@
 
 This document provides guidelines how to make a cellular device adaptation for Mbed OS.
 
+**Note:** Application developers should use only the `API` folder where Cellular public API is defined. All other Cellular APIs are considered as internal APIs and can change in between any release. If modem vendors want their drivers to be maintained for internal API changes, they need to contribute their drivers to mbed-os/master.
```

Comment:
I note this would mean that the proposed `get_at_handler()->get_file_handle()` in the proposed deprecation of `CellularDevice::get_file_handle()` would be a redirection to a non-public API.

Reply:
Good point, `ATHandler` should probably be part of public API.